about.html

```diff
@@ -0,0 +1,124 @@
+          <div class="mobile-menu-btn">
+            <button type="button" class="btn">
+              <img src="images/mobile_menu.png" alt="" />
+            </button>
+          </div>
+          <div class="logo">
+            <a href="index.html"><img src="images/logo.png" alt="" /></a>
+          </div>
+          <section class="menus">
+            <ul class="nav-menu">
+              <li><a href="#highlights">Highlights</a></li>
+              <li id="live"><a href="#">presentation</a></li>
+            </ul>
+          </section>
+        </article>
+      </nav>
+    </header>
+    <main>
+      <section class="main">
+        <div class="greeting">"Hello, Sharing world!"</div>
+        <div class="title">
+          COMPUTER SCIENCE <br />
+          CONFERENCE 2022
+        </div>
+        <div class="message">
+          A fun festival where people from more than 80 countries who believe in
+          the value of sharing and openness and create positive change gather
+          and share ideas will be held in Cyprus in December.
+        </div>
+        <div class="contact-us">
+          <p>
+            Feel free to reach out to us for additional informations <br />
+            about the Computer Science Conference 2022
+          </p>
+          <a href="https://telegram.org/"
+            ><i class="fa-brands fa-telegram"></i> </a
+          ><a href="https://www.whatsapp.com/"
+            ><i class="fa-brands fa-whatsapp"></i></a
+          ><a href="https://www.viber.com/en/"
+            ><i class="fa-brands fa-viber"></i
+          ></a>
+          <p>+99999999999</p>
+        </div>
+      </section>
+    </main>
+    <section id="highlights">
+      <h2>Computer Science Conference 2021</h2>
+      <p>Highlights from the last Computer Science Conference.</p>
+      <img src="images/conf-1.jpg" alt="" /><img
+        src="images/conf-2.jpg"
+        alt=""
+      /><img src="images/conf-3.jpg" alt="" /><img
+        src="images/conf-4.jpg"
+        alt=""
+      />
+    </section>
+    <footer>
+      <img id="my-logo" src="images/88x31.png" alt="" />
+      <section>
+        <span
+          ><img src="images/logo_cc_w_147x35.png" alt="" />
+          <img src="images/logo_cck_w_208x35.png" alt=""
+        /></span>
+        <article>
+          Some Rights Reserved. <br />
+          <br />
+          Unless otherwise specified, all content on the website is available
+          under a Computer Science Conference 2.0 Cyprus License. Please see
+          <div class="red-link">
+            https://amexabee.github.io/Capstone-Project/
+          </div>
+          for detailed usage function details and permission conditions .
+          Computer Science Conference is committed to providing a
+          harassment-free meeting experience for all.
+          <span class="red-link">See our Code of Conduct</span> and
+          <span class="red-link">Non-Infringement Policy</span>.
+        </article>
```

Comment:
- [ ] Kindly make sure these elements are well positioned in your foot section bother on mobile and desktop versions as they are on the provided template.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/197214440-b741ee55-0817-4f57-acb3-7256e089b850.png)
Template design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/197214628-fc76b82a-9a47-4e3d-baa7-8e4cc911e74e.png)